pacstall

```diff
@@ -31,6 +31,8 @@ export SRCDIR="/tmp/pacstall"
 export STGDIR="/usr/share/pacstall"
 export STOWDIR="/usr/src/pacstall"
 
+export user=$(logname 2> /dev/null || echo "${SUDO_USER:-${USER}}")
```

Comment:
Shouldn't it be capitalized and something else (so as to not conflict with `USER`), it's a constant after all.